src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTYResources_de.java

```diff
@@ -322,7 +322,7 @@ public Object[][] getContents() {
         {"Unable to set", "{0} kann nicht festgelegt werden: {1}"},
         {"Unexpected event type", "Unerwarteter Ereignistyp: {0}"},
         {"unknown", "unbekannt"},
-        {"Unmonitoring", "\u00DCberwachung von {0} aufheben"},
+        {"Unmonitoring", "Monitoring von {0} aufheben"},
```

Comment:
The English entry is:

        {"Unmonitoring", "Unmonitoring {0} "},

But the German entry now says "Monitoring". I'm  sure what the original "\u00DCberwachung" translates to, other than berwachung is monitoring. Now this resource is partly in English, and is the incorrect English.

Reply:
I don't know German, but when plugging `"Monitoring von {0} aufheben"` into google translate, I get `Unmonitoring` as well. Maybe `Monitoring` is a word that could also be used in German? No clue. The translation tool decided to make this change.

@sormuras do you have any input on this?

Reply:
It's correct.

The "Un" of "Unmonitoring" is moved to the associated verb in German: "aufheben". which in this context translates to "cancel" or "revoke". This seems to be one of the few cases where there's no commonly used single word in German for "Unmonitoring".

> Now this resource is partly in English, and is the incorrect English.

That's how Tech-German often works - take a word from English and use it as-is, sometimes missing the original meaning. Such words or phrases are then referred to as: Denglisch.

In this case "Überwachung" was German, "Monitoring" is Denglisch. I'd revert that to `\u00DCberwachung` - but both are okay.